### PR TITLE
adding release drafter github action 

### PR DESCRIPTION
**Pull request by reetasingh**
## What does this PR do?
Address #257
Adding GitHub workflow which creates draft release notes
Adding a config file for this

**Could not find a way to automatically update version in policy_sentry/bin/cli.py as described in the issue**

[//]: # (Required: Describe the effects of your pull request in detail. If
multiple changes are involved, a bulleted list is often useful.)

## What gif best describes this PR or how it makes you feel?

Alt Text

[//]: # (Encouraged: Insert an appropriate gif/meme to brighten up your reviewer's day.)

## Completion checklist

- [x] CHANGELOG.md has been updated to reflect the changes
- [ ] Additions and changes have unit tests
- [ ] [Unit tests, Pylint, security testing, and Integration tests are passing.](https://github.com/salesforce/policy_sentry/actions). GitHub actions does this automatically
- [x] The pull request has been appropriately labeled using the provided PR labels

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Backend methods:
   * Added some utility functions (`get_statement_from_policy_using_sid`, `get_sid_names_from_policy` to make it easier to future-proof unit tests that rely on the ever-changing AWS IAM data.
   * Bug fix for `get_actions_for_service` (Fixes #245)
+* adding release drafter as github actions (Fixes #257)
```
Comment (reetasingh) [inline CHANGELOG.md:11]: @kmcquade with this change will CHANGELOG.md still be needed?

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
+      - uses: release-drafter/release-drafter@v5
+        with:
+          # (Optional) specify config name to use, relative to .github/. Default: release-drafter.yml
+          # config-name: my-config.yml
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (reetasingh) [inline .github/workflows/release-drafter.yml:19]: I am not sure what should come here

File: .github/release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
```
Comment (reetasingh) [inline .github/release-drafter.yml:5]: with all the labels in place, our PRs should be labelled accordingly

Comment (reetasingh) [reply .github/release-drafter.yml:5]: As a contributor, I do not have permissions to assign labels to my PR currently

Comment (kmcquade) [reply .github/release-drafter.yml:5]: @reetasingh you should have permissions to add labels now

**Comment by kmcquade** [comment]
@reetasingh have you tried this GitHub action in a separate repository for testing purposes?

**Comment by reetasingh** [comment]
> @reetasingh have you tried this GitHub action in a separate repository for testing purposes?

Good idea. I will try that

**Comment by reetasingh** [comment]
@kmcquade tested it on local repo https://github.com/reetasingh/learn_github_actions/blob/main/.github/workflows/release-drafter.yml 

output : https://github.com/reetasingh/learn_github_actions/releases

File: .github/workflows/release-drafter.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+name: Release Drafter
+
+on:
+  push:
```
Comment (reetasingh) [inline .github/workflows/release-drafter.yml:4]: still not sure of this part

File: .github/workflows/release-drafter.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+    branches: [ main ]
```
Comment (kmcquade) [inline .github/workflows/release-drafter.yml:5]: Please change this to main, master (we still use the master branch right now and haven't moved to main)

**Comment by kmcquade** [comment]
@reetasingh thanks! Please add the path to the markdown file to the mkdocs.yml file so it is included in the documentation. That specific part of the mkdocs.yml file for the contributing section can be found here: https://github.com/salesforce/policy_sentry/blob/master/mkdocs.yml#L74

When you add that line, I can go ahead and merge :)

**Comment by reetasingh** [comment]
> @reetasingh thanks! Please add the path to the markdown file to the mkdocs.yml file so it is included in the documentation. That specific part of the mkdocs.yml file for the contributing section can be found here: https://github.com/salesforce/policy_sentry/blob/master/mkdocs.yml#L74
> 
> When you add that line, I can go ahead and merge :)

updated it

**Review by kmcquade — APPROVED** [review]

**Pull request merged**